libtenzir/src/logger.cpp

```diff
@@ -264,10 +264,22 @@ bool setup_spdlog(bool is_server, const tenzir::invocation& cmd_invocation,
     file_sink->set_pattern(file_format);
     sinks.push_back(file_sink);
   }
+  auto overflow_policy = caf::get_or(cfg_file, "tenzir.log-overflow-policy",
+                                     defaults::logger::overflow_policy);
```

Comment:
There's related documentation at https://docs.tenzir.com/setup-guides/tune-performance#logging which should probably be updated to reflect the new default behavior.

Reply:
We decided to make this option hidden and remove it again after the dust settled.